### DISCUSSION TITLE
Firmwareaktualisierung auf 23.6.21

### DISCUSSION THREAD

**Discussion by dajo77** [tbnobody/OpenDTU#1075] [Q&A]
Ich habe eben versucht von der Version 23.6.14 auf 23.6.21 zu aktualisieren.
Auch nach ca. 5 Min. Wartezeit bleibt die Hochlade-Anzeige bei 100% und dann passiert nix.
Es ist ein ESP-32S Modul mit angeschlossenem 1.3 Zoll Display und eine Funkplatine NRF24L01+ mit "langer" Antenne.

Jemand eine Idee woran es liegen könnte ?

**Comment by teletapi** [comment]
falsche Firmware hochgeladen...?

**Comment by dajo77** [comment]
Habe die Dateien mehrfach heruntergeladen und getestet, ohne Erfolg !
Befürchte, es bleibt mir nur noch Factory Reset, oder ?

**Comment by dajo77** [comment]
Das Problem hat sich gelöst, habe von einem anderen Rechner probiert; und seltsamerweise funktioniert es von da aus !

**Reply by DPO99** [reply]
ich habe das auch schon beobachtet. geholfen hat auch ein restart der opendtu. hängt sicher iwie an nem cache oder sowas.

**Comment by teletapi** [comment]
trotzdem bleibt die Frage unbeantwortet welche Datei du geladen und geflasht hast......

aber ist ja schön das es jetzt funktioniert

**Comment by dajo77** [comment]
Ich habe opendtu-generic.bin in der Version 23.6.21 genommen; und es hat geklappt !
